docs/Installation.md

```diff
@@ -179,8 +179,8 @@ Another option is to use a desktop editor that supports editing files
 over the "scp" and/or "sftp" protocols. There are freely available tools
 that support this (eg, Notepad++, WinSCP, and Cyberduck).
 Load the printer config file in the editor and then save it as a file
-named "printer.cfg" in the home directory of the pi user
-(ie, /home/pi/printer.cfg).
+named "printer.cfg" under `~/printer_data/config`
+(ie, /home/pi/printer_data/config/printer.cfg).
```

Comment:
Probably best to word it so that the user knows that pi may not be the username

Thanks
James

Reply:
Makes sense. Pushed an update.